boards/arm/particle_argon/board.cmake

```diff
@@ -3,9 +3,9 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
+# board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
 board_runner_args(nrfjprog "--nrf-family=NRF52")
 board_runner_args(jlink "--device=nrf52" "--speed=4000")
-include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+# include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
```

Comment:
Same here?

Reply:
Its for convenience as I you nrfjprog all the time.
Can revert